cost-analyzer/templates/cost-analyzer-pvc-template.yaml

```diff
@@ -1,6 +1,7 @@
 {{- if .Values.persistentVolume -}}
 {{- if not .Values.persistentVolume.existingClaim -}}
 {{- if .Values.persistentVolume.enabled -}}
+{{- if and .Values.kubecostDeployment.statefulset.enabled .Values.kubecostDeployment.leaderFollower.enabled -}}
```

Comment:
Don't think we want to force users to run a statefulset to run a PV

Reply:
good catch @AjayTripathy, before the changes proposed to be done for a better combination with `.Values.kubecostDeployment.leaderFollower.enabled` it was a negated `.Values.kubecostDeployment.statefulSet.enabled` so I added `(not .Values.kubecostDeployment.statefulSet.enabled)` instead